src/convertToObject.js

```diff
@@ -1,12 +1,40 @@
 'use strict';
 
+/**
+ * @param {string} sourceString
+ *
+ * @return {object}
+ */
+'use strict';
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const parts = sourceString.split(';');
+
+  parts.forEach((str) => {
+    const cleanStr = str.trim();
+    const colonIndex = cleanStr.indexOf(':');
+
+    if (colonIndex === -1) {
+      return;
+    }
+
+    const property = cleanStr.slice(0, colonIndex).trim();
+    const value = cleanStr.slice(colonIndex + 1).trim();
```

Comment:
You can simplify it
```suggestion
    const [property, value] = str.split(':').map((item) => item.trim());
```

Reply:
I see you didn't apply the suggested changes
If it was unclear what exactly the suggested code does it's always better to ask it in the chat (or you have any other questions)

Generally, here you can use `split` method instead of code you've written
`const [property, value] =` - this part is destructuring assignment